randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+            "Metroid Quarantine A without Space Jump Boots, Morph Ball Bombs, or Spider Ball."
+        ),
+    ],
+    "4176439895": [
+        # Frozen Hunter Metroid Hop Tip
+        # Replaces Hunter Metroid scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Hunter Metroid",
+        (
+            "&main-color=#89D6FF;Jumping on the center of a frozen Hunter Metroid causes one to bounce on top of it."
+            "\n\nWhen it breaks out of the ice, anyone on top is launched high in the air.\n\n"
+            "This can be used to acquire Frost Cave's item without the Grapple Beam."
+        ),
+    ],
+    "425486343": [
+        # Arboretum door DBJ tip
+        # Replaces Arboretum big branch scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;It is possible to surmount the gate "
+            "at the top of this room with a carefully-timed double bomb jump."
+        ),
+    ],
+    "1102487982": [
+        # FCS climb tip
+        # Replaces Frigate Crash Site frigate scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;It is possible to climb up to this room's top door from the bottom. "
+            "There are multiple methods, the easiest of which only needs the "
+            "&main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; to climb the crashed frigate's exterior."
+        ),
+    ],
+    "3737672425": [
+        # Monitor Station climb tip
+        # Replaces Monitor Station 2f left computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;By jumping along the rectangular extrusions on this building's exterior, "
+            "it is possible to reach the roof without Space Jump Boots."
+        ),
+    ],
+    "3893588753": [
+        # Monitor Station climb tip
+        # Replaces Monitor Station 2f middle computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;While on this building's roof, it is possible to reach Warrior Shrine without "
+            "lowering the bridge. To do so, stand on top of the roof computer's tallest monitor, "
+            "lock onto a Puffer to the right or left, and perform a lock-on dash towards the ledge."
+        ),
+    ],
+    "1839087853": [
+        # Tree Weakness Tip
+        # Replaces Main Plaza tree scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Opening the tree requires a &main-color=#FF3333;Super Missile&main-color=#89D6FF;."
+            "\n\n\nAdditionally, it is possible to peek the item inside the tree without opening it. "
+            "To do so, stand at a specific angle and location on the right side of the tree's lip, then jump in place."
```

Comment:
This isn't a trick and it replaces scan text that conveys information about the vulnerability in the tree, so I'd either remove this scan or find a different node to use.

Reply:
This replacement provides more specific information about how to open the tree, so I'm surprised you think it's worth removing. Not saying you're wrong, I'm just confused.